doc/man3/BIO_f_ssl.pod

```diff
@@ -241,6 +241,16 @@ a client and also echoes the request to standard output.
      exit(1);
  }
 
+ /*
+  * Added BIO_do_accept call: the first BIO_do_accept made the socket bind
```

Comment:
This comment needs revising. Please remove "Added BIO_do_accept cal: l" - it doesn't seem to make sense here. The rest of the sentence is fine.